examples/game-clock/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
-    kotlin("jvm") version Version.kotlin
+    `jaicf-kotlin`
```

Comment:
Maybe it's better not to use our own plugins inside examples? It's a place people may copypaste gradle build configuration from.